homeassistant/components/statistics/sensor.py

```diff
@@ -364,7 +364,7 @@ def __init__(
 
         self.states: deque[float | bool] = deque(maxlen=self._samples_max_buffer_size)
         self.ages: deque[datetime] = deque(maxlen=self._samples_max_buffer_size)
-        self.attributes: dict[str, StateType] = {}
+        self._attr_extra_state_attributes = {}
```

Comment:
Is there a reason why you removed the typing info?

Reply:
It's already typed by the `Entity` it inherits from.